pkg/metrics/otel.go

```diff
@@ -121,6 +121,28 @@ processors:
           - cluster_scoped_resource_count
           - resource_ns_count
           - api_duration_seconds
+  # Transform metrics sent to Cloud Monitoring to aggregate high-cardinality labels (e.g. "commit")
+  metricstransform/cloudmonitoring:
+    transforms:
+      - include: last_sync_timestamp
+        action: update
+        operations:
+          - action: aggregate_labels
+            label_set: [status]
```

Comment:
This is the array of labels that you want to keep after the aggregation, have you tried this ConfigMap on a current installed CS to see if labels like `configsync.sync.*` still remain?

Reply:
I did not. I'll add those. Are there others added we need to preserve?

Reply:
Verified that commit is the only missing label now.
![Screenshot from 2023-08-29 15-39-48](https://github.com/GoogleContainerTools/kpt-config-sync/assets/3980984/0e94e932-5309-4a69-ae77-fab914dbafc5)